tuneit/class_utils.py

```diff
@@ -222,9 +222,11 @@ def args_to_kwargs(cls, *args):
 
         return kwargs
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, var_name=None, *args, **kwargs):
```

Comment:
`var_name` should be put after `*args`.

Then shall we have also a parameter `name` that sets the name of the node?
This would also requires a property `name` that should be used instead of `__name__` everywhere.
Finally in the example, we can use `name="sparse"`, for i) obtaining a better label than `bsr` for the node and ii) it would replace the usage of `var_name` since we would get directly `which_sparse`.